data/schedules_report/monthly.yaml

```diff
@@ -0,0 +1,18 @@
+name: testing schedule 1
+description: description of testing schedule 1
+filter:
+    - Events
+    - Policy
+    - Policy Events for Last Week
```

Comment:
I think the policy event of all reports can also  be updated to match the time .

"Policy Events for Last Month "

Reply:
@sshveta In report field there is no  "Policy Events for Last Month " option available, that's why i added for last week.

Reply:
:+1: